packages/tools/devtools/devtools-view/src/components/AudienceHistoryTable.tsx

```diff
@@ -3,28 +3,28 @@
  * Licensed under the MIT License.
  */
 
-import React from "react";
 import {
-	tokens,
+	Table,
 	TableBody,
 	TableCell,
-	TableRow,
-	Table,
 	TableHeader,
 	TableHeaderCell,
+	TableRow,
 	makeStyles,
+	tokens,
```

Comment:
Is there an option to ignore case when sorting? This is better than before, but if we can get everything in simple alphabetical order it'd be better IMO.

Reply:
No option for that. That's either a good thing or a bad thing depending on your perspective. :)